hoth_api.yaml

```diff
@@ -1,62 +1,90 @@
 openapi: 3.0.2
 info:
   description: >-
-    This is the ATAT Provisioning API, an internal API solely meant for use by the ServiceNow component of ATAT in
-    order to kick off provisioning jobs.
-  version: '0.1'
-  title: ATAT Internal API - Provisioning
+    This is the HOTH (Hyperscaler Orchestration and Tracking Helper) API, a private internal-facing API that is intended
```

Comment:
```suggestion
    This is the HOTH (Hyperscale Orchestration and Tracking Helper) API, a private internal-facing API that is intended
```

Reply:
Not according to the README :) We're orchestrating and tracking the hyperscalers. We're not hyperscale

Reply:
We can, of course, change the README. But regardless, gotta be consistent.

Reply:
Haha fair enough, I just assumed it was a typo. You win :)